config/locales/pt-BR.yml

```diff
@@ -33,4 +33,5 @@ pt-BR:
   ended_message        : "O Maior São João do Mundo 2016 acabou! Não há informações futuras disponíveis."
 
   error_invalid_days   : "O Maior São João do Mundo acontece de 03 de Junho a 03 de Julho. Informe um dia válido para obter maiores informações."
-  error_after_end_date : "Apenas datas entre o intervalo de 03 de Junho a 03 de Julho e maiores que a data atual são consideradas válidas."
\ No newline at end of file
+  error_after_end_date : "Apenas datas entre o intervalo de 03 de Junho a 03 de Julho e maiores que a data atual são consideradas válidas."
+  error_invalid_weekday: "Por favor, forneça um dia da semana válido (lembre-se de usar apenas as três primeiras letras (exemplo: seg))."
```

Comment:
If you fix this according to the previous comment, don't forget to change this too.